components/api_request_helper/api_request_helper.h

```diff
@@ -58,6 +58,7 @@ class APIRequestResult {
   const std::string& body() const { return body_; }
   // `base::Value` of sanitized json response.
   const base::Value& value_body() const { return value_body_; }
+  base::Value& value_body() { return value_body_; }
```

Comment:
Added this so I can move `value_body` to another thread, rather than cloning it.

Reply:
Yep, it's also an important thing.
I have a PR to fix that problem more globally: https://github.com/brave/brave-core/pull/21876/files#diff-ecc8fbcb6cb29a6a68559a53dbfe02bdf099789e5df43bfef140084cb010c156

Reply:
Sweet, that should help things heaps :smile: Lazily getting the string should help quite a bit.